### PR TITLE
List all common word/data between two cases

### LINKED ISSUE #198: Maximum common word/data between two cases.
**Is your feature request related to a problem? Please describe.**
Maximum common word/data between two cases.
Create an API that will return a word which is common between two cases.

### LINKED ISSUE #199: List all common word/data between two cases
**Is your feature request related to a problem? Please describe.**
List all common word/data between two cases
Create an API that will return a list of all common words between two cases.

### PR DESCRIPTION
**Pull request by swapnalshahil**
# Description

List all common word/data between two cases
Create API which will give all the common words between two cases. (this will help in the front-end part)
API : http://127.0.0.1:5000/common/case1/case2
method : ["POST"]
{
        "case1": "case_name_one",
        "case2": "case_name_two"
}

Fixes #199 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

POSTMAN

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by swapnalshahil** [comment]
@shivanshu1333 sir! Please review!

**Comment by swapnalshahil** [comment]
# Description

Maximum common word/data between two cases.
This API is responsible for getting common words with their frequencies between the two cases present in the database. I will use this API as per need(depends how much I want to show ) when working on the front end.
API : http://127.0.0.1:5000/common/words/case1/case2
method : ["POST"]
{
        "case1": "case_name_one",
        "case2": "case_name_two"
}

Fixes #198 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

POSTMAN







# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

**Review by swapnalshahil — COMMENTED** [review]
I added this week's commit in the same PR.

File: flask-backend/api/routes/commonwords.py [hunk 1]
```diff
@@ -0,0 +1,412 @@
+'''
```
Comment (shivanshuraj1333) [inline flask-backend/api/routes/commonwords.py:1]: Logic is appropriate.
Nit: format the code properly, remove redundant line spaces.

Comment (swapnalshahil) [reply flask-backend/api/routes/commonwords.py:1]: Done @shivanshu1333  sir!

**Review by shivanshuraj1333 — APPROVED** [review]

**Pull request merged**